IBF-Typhoon-model/lib/pipeline.py

```diff
@@ -0,0 +1,314 @@
+#!/bin/sh
+
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Oct 31 16:01:00 2020
+
+@author: ATeklesadik
+"""
+import sys
+import os
+from datetime import datetime, timedelta
+from sys import platform
+import subprocess
+import logging
+import traceback
+
+import pandas as pd
+from pybufrkit.decoder import Decoder
+import numpy as np
+from geopandas.tools import sjoin
+import geopandas as gpd
+import click
+
+# Set up logger
+logging.root.handlers = []
+logging.basicConfig(format='%(asctime)s : %(levelname)s : %(message)s', level=logging.DEBUG, filename='ex.log')
```

Comment:
Could we put `name` back in the logging messages (at least for the output file)? I find it's pretty useful to have for debugging